halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/OrcaProfileFactory.java

```diff
@@ -80,14 +81,18 @@ protected void setProfile(
     // For backward compatibility
     profile.appendContents("pipelineTemplate.enabled: " + pipelineTemplates);
 
+    // Plugins
```

Comment:
Comments like this generally make me think the logic that follows them should be extracted to a function.